docs/data/data-grid/recipes-editing/recipes-editing.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Data Grid - Editing recipes
+---
+
+# Data grid - Editing recipes
```

Comment:
It's great to have more content!

Now, are we sure these "features" are best documented in a misc section, called "Recipes"?

Looking at each demo, as a developer, I think that I would expect each of these features to be covered in side the "Editing" part of the docs. For example, AG Grid has many pages about editing: https://www.ag-grid.com/javascript-data-grid/undo-redo-edits/. It seems that we draw a line between, the feature is built-in vs. the feature should be composed but isn't a feature that needs to be composed as relevant as one that is built-in?

Reply:
> Looking at each demo, as a developer, I think that I would expect each of these "features" to be covered in side the "Editing" part of the docs.

Adding more content to the page creates a different problem - pages are too long and we need to split them at some point. Offloading custom examples to Recipes reduces this problem.

> It seems that we draw a line between the feature is built-in vs. the feature is composed

For me it's more like "Essential" vs "Advanced".

> either the demo is in the docs, maintained like any other features, or it's not, it's something we quickly put together on a codesandbox with pinned dependencies

I think quick demo is still an option, but for popular customizations use cases it's better to have it maintained in the docs.
I feel like Recipes in this case fill the gap between codesandbox demos and docs - the demo is not important enough to increase the scope of the feature page, but is important enough to have it maintained.

Reply:
> Offloading custom examples to Recipes reduces this problem.

@cherniavskii I agree with this.
My question is more on: where should it be located (navigation tree + URL)?
I took the editing feature as an example. https://mui.com/x/react-data-grid/editing/ is one big page. If we use https://www.notion.so/mui-org/Technical-writing-7e55b517ac2e489a9ddb6d0f6dd765de#075e05fa0a934b59a5b0bded25dea935, then point 1. and 2. are asking us to break down the page. At this point, we might be able to move the content closer to the documentation of the feature. We can move https://mui.com/x/react-data-grid/recipes-editing/ to a recept editing page under the main "Editing" navigation 😁

> For me it's more like "Essential" vs "Advanced".

Fair enough.

> fill the gap between

Fair enough.

Reply:
So anyone interested/resonates with the idea of splitting https://mui.com/x/react-data-grid/editing/ into smaller pages 😁? Like @flaviendelangle did for #5195

Reply:
> As for pages that are not split yet, I assume we would just add Recipes section and if page grows too much then split?

@cherniavskii It sounds great.

> would it make sense to split and have another level of nesting

I don't know but it's not unheard of. AG Grid goes to 4 levels of depth with the docs of the data grid https://ag-grid.com/javascript-data-grid/grouping-single-group-column/. We go up to 3 levels. However in our case, the first level is the data grid component, in their case, they don't even have it. So it's like we were going 5 level deep, while we go down up to 3 right now. Personally, I think that AG Grid should flatten more